tests/Unit/AcceleratorViewCopy/common.h

```diff
@@ -11,6 +11,12 @@
     printf ("error: TEST FAILED\n%s", KNRM );\
     abort();
 
+#define HostToDeviceCopyTest 0x1
+#define DeviceToDeviceCopyTest 0x2
+#define DeviceToHostCopyTest 0x4
+#define HostToDeviceAsyncCopyTest 0x18
+#define DeviceToHostAsyncCopyTest 0xFF
+
```

Comment:
Hi @scchan , thanks for pointing it out. It's actually not a bug, as it won't affect any units of the two avstress tests as what I've tested locally. But I've changed them back, for your concern here. Please review.